### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25579,6 +25591,7 @@ wikpedia->wikipedia
 wil->will, well,
 wilcard->wildcard
 wilcards->wildcards
+wile->while
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `wile` is a somewhat rare but not totally obscure real word

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:25594]: disabled then ? I see it frequently in https://github.com/search?q=wile&type=Wikis

**Comment by larsoner** [comment]
@luzpaz can you rebase or close and open a fresh one now that the dictionary has a `dictionary_rare.txt` where you can put the `wile` change?

**Comment by luzpaz** [comment]
@larsoner done in 32b550e

**Pull request merged**